website/docs/config/networks.mdx

```diff
@@ -71,3 +67,38 @@ To target the `ghostnet` network we configured above using the `testing` environ
 ```shell
 taq originate <contractName> --env testing
 ```
+
+## Deploying and Transferring with Mainnet
+
+Configuring a mainnet is quite similar to configuring a teztnet with one difference. Users must also create an account, with the mnemonic specified, within the network in order to interact with mainnet. See the example below:
+
+:::danger
+Improvements are on the way. However, we currently only support signing in memory and we require a mnemonic to be written in the config file, so do not commit this file or delete the mnemonic before doing so). This is not the safest way to interact with mainnet so proceed with caution
```

Comment:
Missing opening bracket

Reply:
Will fix!

Reply:
Forgot to delete closing bracket actually